BookMyShowApp.API/Interface/ICustomer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using BookMyShow.Data;
+using BookMyShow.Data.ViewModels;
+
+namespace BookMyShowApp.API.Interface
+{
+    public interface ICustomer
+    {
+        Task<IEnumerable<Customer>> SaveCustomerDetails(CustomerModel c);
```

Comment:
1. Give proper name for input parameter
2. What it is returning IEnumerable<Customer> ?